### PR TITLE
CW-18 - User interface changes

### PR DESCRIPTION
**Pull request by Tonyynot14**

### CODE REVIEW

File: front/src/containers/SearchPage/SearchView2.tsx [hunk 1]
```diff
@@ -396,7 +396,7 @@ class SearchView2 extends React.Component<SearchView2Props, SearchView2State> {
               style={{ margin: '5px', color: 'gray' }}
             />
           )}
-        <div>{recordsTotal} results</div>
+
```
Comment (kaveer-k) [inline front/src/containers/SearchPage/SearchView2.tsx:399]: Delete empty line

**Review by kaveer-k — CHANGES_REQUESTED** [review]
Mostly code formatting issues

File: front/src/containers/SearchPage/SearchPage.tsx [hunk 2]
```diff
@@ -801,10 +808,12 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
 
     return (
       <SearchContainer>
-        <InstructionsContainer>
+        <InstructionsContainer  >
           {presearchText && (
             <Instructions>
               <RichTextEditor
```
Comment (kaveer-k) [inline front/src/containers/SearchPage/SearchPage.tsx:814]: Do instructions even need an editor?

File: front/src/containers/SearchPage/SearchView2.tsx [hunk 3]
```diff
@@ -891,7 +902,8 @@ class SearchView2 extends React.Component<SearchView2Props, SearchView2State> {
             key="default"
             id="dropdown-basic-default"
             style={{
-              margin: '1em 1em 1em 0',
+              margin: '0em 1em 0em 1em',
+              width: "200px",
```
Comment (kaveer-k) [inline front/src/containers/SearchPage/SearchView2.tsx:906]: Looks like we're using single quotes for styles so would recommend keeping it consistent.

**Review by kaveer-k — CHANGES_REQUESTED** [review]

File: front/src/containers/SearchPage/SearchPage.tsx (hunk 2 above)
Comment (Tonyynot14) [reply front/src/containers/SearchPage/SearchPage.tsx:814]: Yeah they are using it to add in the presearch instructions into the page. I think they originally wanted to be able to use markup for this in particular.

File: front/src/containers/SearchPage/SearchPage.tsx [hunk 4]
```diff
@@ -161,13 +161,20 @@ const SearchContainer = styled.div`
   display: flex;
   flex-direction: column;
   padding: 10px;
+  .RichTextEditor__root___2QXK-{
+    background-color: inherit !important;
```
[the remark this answers is not in the document]
Comment (Tonyynot14) [reply front/src/containers/SearchPage/SearchPage.tsx:165]: We want the theme color that is used for the background used right here. I am working on making the color that color right now but not sure of the correct color to access

File: front/src/containers/SearchPage/SearchPage.tsx (hunk 2 above)
Comment (kaveer-k) [reply front/src/containers/SearchPage/SearchPage.tsx:810]: Ah I see, thanks for clarifying.

**Review by kaveer-k — CHANGES_REQUESTED** [review]
Hey @Tonyynot14, thanks for cleaning up the code formatting and other issues. There's a few UI/UX things I noticed. Will comment on the JIRA ticket and add some mockups.

**Review by kaveer-k — CHANGES_REQUESTED** [review]
@Tonyynot14 Can you please resolve the merge conflicts?

**Review by kaveer-k — APPROVED** [review]

**Pull request merged**